.test-infra/jenkins/job_PostCommit_Go_GradleBuild.groovy

```diff
@@ -37,6 +37,9 @@ PostcommitJobBuilder.postCommitJob('beam_PostCommit_Go_GradleBuild', 'Run Go Pos
       tasks(':goPostCommit')
       commonJobProperties.setGradleSwitches(delegate)
       switches('--no-parallel')
+
+      // Pre-commits run in a separate Jenkins job
+      switches('--exclude-task :goPreCommit')
```

Comment:
Though, could it just be that `:goPostCommit` should be modified? At some point I thought there was an idea to keep the Jenkins scripts mostly static and do all the tweaking in gradle, since then you don't have to run the seed job and whatnot to test changes.

Reply:
Yes, I think it would be better if the task encapsulated as much of the execution description as possible, rather that tacking on extra command-line options. This also makes it easier to see everything in one place and straight-forward to execute locally.

The reason I didn't do that in this change was in case there is an implicit assumption that "post-commit" means "run all the tests"; with this change you'd need to run pre-commit + post-commit. (I'll also note that there's been previous discussion to improve our test suite naming scheme)

I'm going to make the change as you propose, and I'll bring it up on the dev@ list as a heads-up and to make sure there are no objections.